src/Monolog/Handler/SlackWebhookHandler.php

```diff
@@ -46,8 +46,9 @@ class SlackWebhookHandler extends AbstractProcessingHandler
      * @param int         $level                  The minimum logging level at which this handler will be triggered
      * @param bool        $bubble                 Whether the messages that are handled can bubble up the stack or not
      * @param array       $excludeFields          Dot separated list of fields to exclude from slack message. E.g. ['context.field1', 'extra.field2']
+     * @param array       $markdownIn             List of fields to use markdown in when using an attachment
```

Comment:
Would be good to have docs here as to which fields are accepted, or maybe even validation in SlackRecord of which fields can be configured if this is a fixed list on slack's end.